src/content/docs/browser/browser-monitoring/browser-pro-features/auto-logging.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: "Auto logging"
+metaDescription: "Get a more complete picture of your front-end application"
+freshnessValidatedDate: 2024-11-18
+---
+
+Auto logging collects log messages issued from the browser `console` to help you maximize observability of your front-end applications.
+
+Browser logs are tracked by default for Pro and Pro+SPA agents, but unavailable for the Lite browser agent. We recommend that you first confirm you're using the Pro or Pro+SPA agent.  See [Getting Started](#get-started).
+
+## How browser auto-logging works [#how-it-works]
+
+Based on logging levels and sampling rates set in the configuration, auto-logging may collect messages from the following:
+- `console.error`
+- `console.warn`
+- `console.info`
+- `console.log` (this will be treated the same as `INFO` level)
+- `console.debug`
+- `console.trace`
```

Comment:
this might benefit from being shown as a table of the console method and the log level that is captured.

method | level
console.error | 'ERROR' 

etc.